packages/miniflare/src/index.ts

```diff
@@ -1529,6 +1576,8 @@ export class Miniflare {
 		if (disposing) return new URL("http://[100::]/");
 		// Make sure `dispose()` wasn't called in the time we've been waiting
 		this.#checkDisposed();
+		// if there is an inspector proxy let's wait for it to be ready
+		await this.#maybeInspectorProxy?.ready;
```

Comment:
Should this go above the `checkDisposed()` call?

Reply:
I think so? honestly I am not sure about the difference, but it's probably a good idea to move it 🙂👍

Reply:
moved 🙂👍